.github/workflows/build.yml

```diff
@@ -5,63 +5,74 @@ on:
       - main
 
 jobs:
-  build:
+  get-version:
     runs-on: ubuntu-latest
-    permissions:
-      contents: write
-      id-token: write
+    outputs:
+      version: ${{ steps.get_version.outputs.version }}
     steps:
       - name: Checkout
         uses: actions/checkout@v4
-        with:
-          fetch-depth: 0
-      - name: Install Node.js
-        uses: actions/setup-node@v4
-        with:
-          node-version: 20
       - name: Get version
         id: get_version
         run: echo "version=$(grep "version" Cargo.toml | head -n1 | cut -d '"' -f2)" >> $GITHUB_OUTPUT
       - name: Compare version to NPM
         run: |
-          NPM_VERSION=$(npm view create-o7-app version)
+          NPM_VERSION=$(curl -s https://registry.npmjs.org/create-o7-app | jq -r '.["dist-tags"].latest')
           MY_VERSION=${{ steps.get_version.outputs.version }}
           if [ "$NPM_VERSION" == "$MY_VERSION" ]; then
             echo "NPM version is the same as the version in Cargo.toml, exiting"
             exit 1
           fi
-      - name: Install pnpm
-        run: npm i -g pnpm
-
-      - name: Generate changelogs
-        run: |
-          # Full changelog
-          pnpx git-cliff --tag ${{ steps.get_version.outputs.version }} -o CHANGELOG.md
-          # Changelog for the release
-          pnpx git-cliff --tag ${{ steps.get_version.outputs.version }} --unreleased --strip header -o CHANGELOG-release.md
 
+  build-linux:
+    needs: get-version
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
       - name: Install Rust
         uses: dtolnay/rust-toolchain@stable
         with:
-          components: rustfmt
-          targets: x86_64-unknown-linux-musl,x86_64-pc-windows-gnu,i686-pc-windows-gnu,x86_64-apple-darwin,aarch64-apple-darwin
-      # - name: Test
-      #   run: cargo test
-      - name: Install MinGW
-        run: |
-          sudo apt-get update
-          sudo apt-get install -y gcc-mingw-w64-x86-64 gcc-mingw-w64-i686
+          targets: x86_64-unknown-linux-musl
+      - name: Cache Rust
+        uses: Swatinem/rust-cache@v2
       - name: Install MUSL
+        shell: bash
```

Comment:
Is bash not the default?

Reply:
I ended up forgetting to remove it here after I removed the actions